kong/plugins/log-serializers/basic.lua

```diff
@@ -21,7 +21,12 @@ function _M.serialize(ngx)
   local request_tls
   local request_tls_ver = ngx_ssl.get_tls1_version_str()
   if request_tls_ver then 
-    request_tls = { version = request_tls_ver } 
+    request_tls = { 
+      version = request_tls_ver, 
+      cipher = var.ssl_cipher,
+      supported_client_ciphers = var.ssl_ciphers,
```

Comment:
Is it truly necessary to log client supported ciphers like so? What is the use-case? This will significantly increase the payload size for those logs. Example running the test locally with OpenSSL 1.1.0j and TLS 1.2:
```
supported_client_ciphers = "ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:
DHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:
DHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:
DHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:
DHE-RSA-AES256-SHA256:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256:
DHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES256-SHA:ECDHE-RSA-AES256-SHA:
DHE-RSA-AES256-SHA:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES128-SHA:DHE-RSA-AES128-SHA:
AES256-GCM-SHA384:AES128-GCM-SHA256:AES256-SHA256:AES128-SHA256:AES256-SHA:
AES128-SHA:0x00ff"
```

TLS 1.3 has a much more restricted list of supported ciphers which would make for more reasonably size payloads, but then one can ask what value logging them would bring, if we can reasonably expect what they are going to be?

Reply:
Understanding in large deployments what’s preventing the adoption of more modern ciphers (old clients). In general Kong should be logging everything and also - when the log size is too long - have a way to dynamically exclude certain fields (but this is out of the scope of this PR).

Reply:
Note that a client can send *hundreds* of supported ciphers: this could quickly overwhelm most logging formats.

Reply:
Okay I have removed `supported_client_ciphers` in my latest commit to speed up merging this PR.